src/services/role/utils/permissions.js

```diff
@@ -20,34 +28,39 @@ const ROLES = {
  * 'opt-in', 'disabled', undefined - removes them from the given role
  *
  * @param {String} env
- * @param {String} role
+ * @param {String[]} roles
  * @param  {...String} permissions
  */
-const definePermissions = (env, role, ...permissions) => {
+const definePermissions = (env, roles, ...permissions) => {
+	// TODO should this not be opt-in? is this vice-versa
 	if (['opt-out', 'enabled'].includes(Configuration.get(env))) {
-		// set defaul permission
-		Role.update({
-			name: role,
-		},
-		{
-
-			$addToSet: {
-				permissions: {
-					$each: permissions,
+		// set default permission
+		roles.forEach((role) => {
+			Role.update(
+				{name: role,},
+				{
+					$addToSet: {
+						permissions: {
+							$each: permissions,
+						},
+					},
 				},
-			},
-		}, { multi: true }).exec();
+				{multi: true}
+			).exec();
+		});
 	} else if (['opt-in', 'disabled', undefined].includes(Configuration.get(env))) {
-		// remove defaul permission
-		Role.update({
-			name: role,
-		}, {
-			$pull: {
-				permissions: {
-					$in: permissions,
+		// remove default permission
+		roles.forEach((role) => {
+			Role.update({
+				name: role,
+			}, {
+				$pull: {
+					permissions: {
+						$in: permissions,
+					},
 				},
-			},
-		}, { multi: true }).exec();
+			}, {multi: true}).exec();
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6051638)